robottelo/hosts.py

```diff
@@ -413,6 +413,16 @@ def install_capsule_katello_ca(self, capsule=None):
         if result.status != 0:
             raise ContentHostError('Failed to install the katello-ca rpm')
 
+    def install_cockpit(self):
+        """Installs cockpit on the broker virtual machine.
```

Comment:
Install cockpit on the Content Host.

(no need for broker virtual machine)

Reply:
Just wanted to use a similar docstring to other functions in this module